integrations/langfuse/src/haystack_integrations/tracing/langfuse/tracer.py

```diff
@@ -64,6 +65,56 @@
 tracing_context_var: ContextVar[Dict[Any, Any]] = ContextVar("tracing_context")
 
 
+def to_openai_dict_format(chat_message: ChatMessage) -> Dict[str, Any]:
```

Comment:
@vblagoje what do you think of making this private for now. Since it's only internal to Langfuse and I think we'd prefer users to still use the `ChatMessage.to_openai_dict_format` in general.